SConstruct

```diff
@@ -1367,6 +1367,8 @@ for libraryName, libraryDef in libraries.items() :
 			# LIBSUFFIXES variable used by the library scanner.
 			pythonModuleEnv["LIBSUFFIXES"].append( pythonModuleEnv.subst( "$SHLIBSUFFIX" ) )
 			pythonModuleEnv["SHLIBSUFFIX"] = ".so"
+		elif pythonModuleEnv["PLATFORM"] == "win32" :
+			pythonModuleEnv["SHLIBSUFFIX"] = ".pyd"
```

Comment:
Does the comment above about `LIBSUFFIXES` not apply on Windows? I seem to remember tearing my hair out tracking that one down, so don't fancy having to do it again...

Reply:
That sounds plausible to this Windows ignoramus, and we have the comment above (in the Mac section) as a hint for where to start looking if that turns out wrong in future.